### ISSUE TITLE
Media Manager Page [Backend - side] Using Tool [A11y]

### ISSUE THREAD

**Issue by adj9** [joomla/joomla-cms#31401]
### Description of the issue
Use Tool: [ARC Toolkit](https://chrome.google.com/webstore/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce) on Chrome.
Test on Media Manager Page.

### Error Results
<b>Structure category - Pseudo content sub-category</b>
- 9 errors with description (visible and hidden alerts errors)

Description:

> Error: pseudo content found on text element, not in Private Use Area
Recommend: add the pseudo content to its own span with aria-hidden='true'.
            Also, add a text alternative when appropriate

These errors from the tool are classified as <b>non PUA Content In Same Text Element</b>.
In doing some research I seem to have understood that it is a mistake due to a lack of a characteristic of CSS.
I'll bring you back a page: [https://www.w3.org/WAI/GL/wiki/Icon_Font_with_an_On-Screen_Text_Alternative](https://www.w3.org/WAI/GL/wiki/Icon_Font_with_an_On-Screen_Text_Alternative)

Lines of code error detect:
- `<li data-access-action="increaseText" class="">Increase Text Size</li>`
- `<li data-access-action="decreaseText" class="">Decrease Text Size</li>`
- `<li data-access-action="increaseTextSpacing" class="">Increase Text Spacing</li>`
- `<li data-access-action="decreaseTextSpacing" class="">Decrease Text Spacing</li>`
- `<li data-access-action="invertColors" title="" class="">Invert Colours</li>`
- `<li data-access-action="grayHues" title="" class="">Grey Hues</li>`
- `<li data-access-action="underlineLinks" title="" class="">Underline Links</li>`
- `<li data-access-action="readingGuide" title="" class="">Reading Guide</li>`
- `<li data-access-action="textToSpeech" class="">Text to Speech</li>`v

### Relevant Warning Result 
- 5 warning detect

Description:
> Warning: pseudo content found
Recommend: ensure text alternative is provided when appropriate

These warning from the tool are classified as <b>pseudo Content Check For Alternative Text</b>.

Lines of code error detect:
- `<span class="icon-images mediamanager" aria-hidden="true"></span>`
- `<span class="icon-joomla" aria-hidden="true"></span>`
- `<span class="icon-folder"></span>`
- `<span class="icon-folder"></span>`
- `<span class="icon-folder"></span>`

The last three warnings refer to the same line, but the Tool does not
distinguish between the three subtypes of warnings.
